.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: Release
+on:
+  push:
+    branches: [main]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.10
+      - name: Install
+        run: npm install
+      - name: Release
+        run: npm run semantic-release
```

Comment:
@patrickocoffeyo I think `semantic-release` by itself normally adds a v in front of the version number so I normally pass the `--tag-format '${version}'` flag to get rid of the v and just keep the version number.  This is more consistent with how the PHP world does version numbers.  I know this is not a strictly PHP repo though, so maybe not applicable.

Reply:
This works for me. I couldn't find any reason to use one or the other.